Dockerfile

```diff
@@ -16,3 +16,6 @@ ADD https://github.com/FFIG/ffig/archive/master.zip /home/ffig/ffig
 
 # Copy in the content of this repository
 COPY . /home/ffig/rest-api
+WORKDIR /home/ffig/flask
+EXPOSE 5000
+RUN python ./run.py
```

Comment:
Also, is RUN the right thing? I thought RUN was to issue commands inside the image while building it; it sounds like this is the entry point when starting a container. I think there's a different command for that?